docs/usage/ruby.md

```diff
@@ -25,3 +25,23 @@ If you always want to have the latest available version, consider specifying `ge
 
 You can install the [Renovate App](https://github.com/apps/renovate) on GitHub.
 Or you can check out [Renovate OSS](https://github.com/renovatebot/renovate) to self-host Renovate.
+
+## Ruby on Rails
+
+Renovate doesn't support [Rails](https://rubyonrails.org/) gems correctly.
```

Comment:
> This is something that's very specific to Rails and as such I'm not sure if you want to handle it. If you do you'd essentially have to scan the `Gemfile` for all Rails gems (that's the ones mentioned here https://github.com/rails/rails#frameworks-and-libraries + `rails` + `railties`) and update all of them whenever a single one needs updating.

As Renovate won't work with Ruby on Rails gems, maybe we should explain how to disable Renovate for those gems? Or create our own `workarounds:ignoreRubyOnRailsUpdates` preset, with the full list of Rails gems?

```json
{
  "packageRules": [
    {
      "description": "Disable Renovate for any Ruby on Rails gems",
      "matchPackageNames": ["list", "all", "ruby", "on", "rails", "gems", "here"],
      "enabled": false
    }
  ]
}
```

Reply:
Yeah adding a link to the docs on how to disable it for certain gems might be helpful here.

Just out of curiosity does renovate deal with these kinds of exceptions to the rules in other package managers at all?

Reply:
I think we have two options:

1. Create and maintain a `workarounds:ignoreRubyOnRailsUpdates` preset with a list of Ruby on Rails gems that Renovate should ignore 
2. Put the example workaround in the docs and let our users create and maintain the list of Ruby on Rails gems

Which do you prefer @rarkins?

Reply:
Another option to consider:

3. Create and maintain a `group:rubyOnRails` preset:
    ```json
    {
      "packageRules": [
        {
          "matchDatasources": [
            "rubygems"
          ],
          "matchPackageNames": [
            "list",
            "all",
            "Ruby on Rails packages",
            "here"
          ],
          "groupName": "Ruby on Rails packages"
        }
      ]
    }
    ```

Reply:
@rarkins which option do you like best? Or maybe you have another idea?

Reply:
`@rarkins` has chosen option 3, see issue:

- #17101